StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -709,6 +725,52 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
         ]
         return post(resource: "consumers/sessions/confirm_verification", parameters: parameters)
     }
+
+    func linkAccountSignUp(
+        requestSurface: String, // "ios_instant_debits"
```

Comment:
This request surface might be rejected right now. I’ll open a backend pull request to add the required enum!

Reply:
Thanks for working on it! PR for context: https://git.corp.stripe.com/stripe-internal/pay-server/pull/864620

Reply:
Merged!